plugins/Agregator/clanky/mytus_o_rakovine.md

```diff
@@ -45,6 +47,8 @@ Z laického pohledu tak vzniká _mýtus_, že viditelné zhoršení zdravotního
 
 ## Rakovina či zcela nezávažné onemocnění
 
+<!-- Comment, REMOVEME -->
```

Comment:
Nadpis `Rakovina či zcela nezávažné onemocnění` mi říká, že je rakovina nezávažné onemocnění. To je účel?